src/main/java/online/book/store/model/Book.java

```diff
@@ -26,4 +32,6 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
+    @Column(nullable = false)
+    private Boolean isDeleted = false;
```

Comment:
I think it is safe to say that we can use a primitive boolean for this column to be sure that we will always have some value there, although in general case boolean fields of entity would likely be wrappers in most cases, so up to you